docs/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM centos/ruby-22-centos7
```

Comment:
I'm curious how we can maintain this file for IOS, Android Cordova?
Do we have separate repository where common files like this one may land?

Reply:
Since we will have couple platforms it will make sense to have one single place for automation (probably the target repository - github pages) where we could place this stuff.